message_store/projections/fetch.py

```diff
@@ -9,39 +9,46 @@ def __init__(self, jetstream_client: JetStreamContext, nats_subject_prefix: str)
         self._jetstream_client = jetstream_client
         self._nats_subject_prefix = nats_subject_prefix
 
-    async def fetch(self, subject: str, projection: Projection):
+    async def fetch(self, subject: str, projection: Projection, until_seq: int = None):        
```

Comment:
Mypy doesn't like `arg: int = None`, consider `arg: int | None = None`. More detail in the failed workflows